tpl/collections/collections.go

```diff
@@ -84,9 +83,9 @@ func (ns *Namespace) After(index interface{}, seq interface{}) (interface{}, err
 	return seqv.Slice(indexv, seqv.Len()).Interface(), nil
 }
 
-// Delimit takes a given sequence and returns a delimited HTML string.
+// Delimit takes a given sequence and returns a delimited string.
 // If last is passed to the function, it will be used as the final delimiter.
-func (ns *Namespace) Delimit(seq, delimiter interface{}, last ...interface{}) (template.HTML, error) {
+func (ns *Namespace) Delimit(seq, delimiter interface{}, last ...interface{}) (string, error) {
```

Comment:
Can you revert this change. It's not directly relevant to the issue in question, and it may also be a possibly breaking change.